manifests/y/Yubico/YubikeyPersonalizationGUI/3.1.25/Yubico.YubikeyPersonalizationGUI.locale.en-US.yaml

```diff
@@ -0,0 +1,16 @@
+# Created using wingetcreate 0.4.4.1
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.0.0.schema.json
+
+PackageIdentifier: Yubico.YubikeyPersonalizationGUI
+PackageVersion: 3.1.25
+PackageLocale: en-US
+Publisher: Yubico

```

Comment:
```suggestion
Publisher: Yubico AB
```
This has to be according to the app list in control panel 
![image](https://user-images.githubusercontent.com/71691514/151668674-e5ab9f13-3fdc-45b7-ae15-3d9c2dff4768.png)